### ISSUE TITLE
Select a range of photos

### ISSUE THREAD

**Issue by whiver** [cozy/cozy-drive#1416]
Hi there :)
Would be great if we had the possibility to select a range of photos in Cozy Photos directly using shift, for example to add a bunch of photo to an album, as for now I have to select each photo individually.
Is there a feature like that that I missed?

Thanks!

**Comment by Crash--** [comment]
Hi! 

Thank you for taking the time to write us something ;). In fact, you already have 2 ways to add a bunch of photos to an album.

First, from the photo view, you just have to hover a photo and you'll see a checkbox at the top left of the photo. You can check it and select several. 

<img width="1229" alt="capture d ecran 2019-01-29 a 08 47 26" src="https://user-images.githubusercontent.com/1107936/51892542-871cda80-23a2-11e9-85a0-193ff186121a.png">


Second, from the album view, when you create an album, you can select multiple photos by checking the checkbox : 
<img width="1154" alt="capture d ecran 2019-01-29 a 08 46 44" src="https://user-images.githubusercontent.com/1107936/51892523-74a2a100-23a2-11e9-80ad-0ec28240936b.png">

**Comment by whiver** [comment]
Thanks for your answer :slightly_smiling_face: 
Indeed, you're right. Though, my concern was about having an easy way to select those photos. Let's say I have 300 photos that I would like to add to an album. I don't want to check each box one by one: it would be easier to have the common shortcut of selecting the first image, holding shift and clicking on the last image to select. Do you see what I mean?

**Comment by Crash--** [comment]
Oh, I see what you mean now. We’ll see what we can do.

**Comment by Gara64** [comment]
Note that you can use the "Tout sélectionner" button on the top-right to select all the photos from a given month. It does not solve your issue, but it is a workaround that might help.

**Comment by whiver** [comment]
Yep, thanks for your answer, unfortunately as I'm currently migrating my data to cozy the photos I'm trying to tag are among the others, so using "select all" would not help :/
